### PR TITLE
Offer escape hatch to remove Sec-WebSocket-Protocol response headers

### PR DESCRIPTION
**Pull request by mjeffrey18**
## Purpose

If you're using non-action-cable JS clients or iOS etc, the Sec-WebSocket-Protocol headers can cause issues.

## Solution

Add env var escape hatch to remove this header from the response.

## Bonus

Add vanilla JS example also

### CODE REVIEW

File: src/cable/handler.cr [hunk 1]
```diff
@@ -14,7 +14,9 @@ module Cable
       path = context.request.path
       Cable::Logger.info "Started GET \"#{path}\" [WebSocket] for #{remote_address} at #{Time.utc.to_s}"
 
-      context.response.headers["Sec-WebSocket-Protocol"] = "actioncable-v1-json"
+      unless ENV["DISABLE_SEC_WEBSOCKET_PROTOCOL_HEADER"]?
+        context.response.headers["Sec-WebSocket-Protocol"] = "actioncable-v1-json"
+      end
```
Comment (jwoertink) [inline src/cable/handler.cr:19]: Since we already include Habitat, what do you think about just making this a Habitat config?

Comment (mjeffrey18) [reply src/cable/handler.cr:19]: Yeah absolutely, that makes sense.

**Review by jwoertink — APPROVED** [review]
🚀

File: examples/non-action-cable-js-client-with-lucky.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<!-- Open in a web browser with Lucky running -->
```
Comment (russ) [inline examples/non-action-cable-js-client-with-lucky.html:1]: This is great! The less JavaScript the better!

**Review by russ — APPROVED** [review]

File: examples/non-action-cable-js-client-with-lucky.html (hunk 2 above)
Comment (fernandes) [reply examples/non-action-cable-js-client-with-lucky.html:1]: why u no like JS?! lol

**Pull request merged**